code/include/env/mhartid.h

```diff
@@ -0,0 +1,8 @@
+#ifndef RISCV_MHARTID_H
+#define RISCV_MHARTID_H
+
+#define MHARTID_HART_ID_SHIFT 0
+
+#define MHARTID_HART_ID_MASK  0xFFFFFFFFFFFFFFFFull
```

Comment:
since mhartid has only 1 field so this file isn't needed

Reply:
Lastly, I want you to deeply understand what each register does. Pls fix by tomorrow.